packages/core-app-api/config.d.ts

```diff
@@ -114,6 +114,11 @@ export interface Config {
      * @visibility frontend
      */
     environment?: string;
+    /**
+     * The 'allowGuestMode' flag for GuestUserIdentity.
```

Comment:
Yep thats correct. The flag focuses on providing a valid token and will not break people's experiences if they don't set it. I defaulted to this wording because it was the one used in the issue comments.